src/pages/steps/ContactInfo.tsx

```diff
@@ -0,0 +1,152 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { css } from 'styled-components';
+
+import { Button, ButtonVariants } from '@/ui/Button';
+import { Icon, Icons } from '@/ui/Icon';
+import { Inline } from '@/ui/Inline';
+import { Stack } from '@/ui/Stack';
+import { Text, TextVariants } from '@/ui/Text';
+import { getValueFromTheme } from '@/ui/theme';
+
+const getValue = getValueFromTheme('contactInformation');
+const borderColorFromTheme = getValue('borderColor');
+const iconColorFromTheme = getValue('iconColor');
+
+const ContactInfoType = {
+  PHONE: 'phone',
+  EMAIL: 'email',
+  URL: 'url',
+} as const;
+
+type ContactPoint = {
+  phone: string[];
+  email: string[];
+  url: string[];
+};
+
+type UrlLabelInLanguages = Partial<{
+  nl: string;
+  fr: string;
+  de: string;
+  en: string;
+}>;
+
+type BookingInfo = {
+  urlLabel?: UrlLabelInLanguages;
+} & { [Property in keyof ContactPoint]?: string };
+
+type Props = {
+  eventContactInfo: ContactPoint;
+  eventBookingInfo: BookingInfo;
+};
+
+const ContactInfo = ({ eventContactInfo, eventBookingInfo }: Props) => {
+  const { t } = useTranslation();
+
+  const mergedContactAndBookingInfo = useMemo(() => {
+    if (!eventContactInfo && !eventBookingInfo) {
+      return { email: [], url: [], phone: [] };
+    }
+    return {
+      email: [...eventContactInfo['email'], eventBookingInfo['email']],
+      url: [...eventContactInfo['url'], eventBookingInfo['url']],
+      phone: [...eventContactInfo['phone'], eventBookingInfo['phone']],
+    };
```

Comment:
This is gonna create new arrays with the eventContactInfo aswel as the eventBookingInfo. But it won't prevent duplicate information. 
If the intent is to have only unique values you could do this:

```typescript
email: [...new Set([...eventContactInfo['email'], eventBookingInfo['email']])]
```

the same for url and phone

Reply:
Good suggestion 👍